test/models/movie.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Movie = require('../../lib/models/movie');
+
+describe('movie model', () =>{
+
+  describe('serialize', () => {
+
+    it('includes all of the necessary fields', () => {
+      const movie = Movie.forge().serialize();
+
+      expect(movie).to.have.all.keys([
+        'id',
+        'title',
+        'release_year',
+        'object'
+        ]);
```

Comment:
nit: I think the `]` is suppose to line up with `expect`, but I'm not 100%. @itsachen ??